README.md

```diff
@@ -307,38 +258,29 @@ docker compose build api
 
 # Building alltogether
 #---------------------------------------------
-docker compose --profile <local|staging|manage> build
+docker compose build
 
 # Run
 #---------------------------------------------
-docker compose --profile <local|staging> up
+docker compose up
 ```
 
-**Note**:  You need to build `morpheus-data` and `morpheus-server` API service (or any other microservice that uses it)
-every time you make a change inside `morpheus-data`. This build is necessary because you need to build the wheel 
-file again and install it in the `morpheus-server` API service or any other service that uses it
-For more information, you can read the morpheus-data [README](./morpheus-data/README.md).
+**Note**: You need to build `morpheus-data` and the `morpheus-server` API service (or any other microservice that uses it) every time you make a change to `morpheus-data`. This is necessary because you need to rebuild the wheel file and install it in the `morpheus-server` API service or any other service that uses it. For more information, see the `morpheus-data` [README](./morpheus-data/README.md).
 
 
-### Adding a new dependency to the backend
+## Adding a new dependency to the backend
 
-1. Add the new dependency directly to the respective `requirements.txt` file
-2. Update the docker image
-   ```shell
-    docker-compose build api
-   ```
-3. Run the image
-   ```shell
-    docker-compose up
-   ```
+1. Add the new dependency directly to the respective `requirements.txt` file.
+2. Update the docker image: `docker compose build api`.
+3. Run the image: `docker compose up`.
 
-**Note:** This project uses requirements.lint.txt only for caching in ci workflow linting job.
+**Note:** This project uses `requirements.lint.txt`` only for caching in CI workflow linting job.
 
-### Adding a new dependency to the frontend
+## Adding a new dependency to the frontend
 
 ```shell
 # Add a new dependency to the npm package.json file
-docker-compose run --rm client yarn install <dependency>
+yarn install <dependency>
 
 # Update the docker image
 docker-compose build client
```

Comment:
@asanchezyali please remove the dash from all parts where you find it, in docker compose:   docker-compose -> docker compose